src/ucs/config/parser.c

```diff
@@ -1222,15 +1276,16 @@ ucs_config_parser_print_field(FILE *stream, const void *opts, const char *env_pr
     ucs_assert(!ucs_list_is_empty(prefix_list));
     head = ucs_list_head(prefix_list, ucs_config_parser_prefix_t, list);
 
-    field->parser.write(value_buf, sizeof(value_buf) - 1, (char*)opts + field->offset,
```

Comment:
if (deprecated) {
 snprintf(value_buf, " (deprecated)"
} else {
 snprintf(value_buf, "=")
 field->parser.write(value_buf +1, ...
}

and remove = from print below

Reply:
good idea, done